include/kafka/ConsumerConfig.h

```diff
@@ -75,6 +75,13 @@ class ConsumerConfig: public Properties
      */
     static const constexpr char* SOCKET_TIMEOUT_MS       = "socket.timeout.ms";
 
+    /**
+     * Control how to read messages written transactionally.
+     * Available options: read_uncommitted, read_committed
+     * Default value: read_committed
```

Comment:
Do we need to follow Java's default value to read_uncommitted?

Reply:
Oh, another mismatched default configuration between `Java` and `librdkafka`? Here's the configuration doc for `librdkafka`, https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
I feels not good about customizing our own `default value` over `librdkafka`, -- there've been a few already, -- I'm considering to remove all of them, in some later days.
Generally, the `modern-cpp-kafka` would follow `Java`'s interfaces, but for these many configuration items, it should just simply forward the config to `librdkafka` (which makes things easier to understand)